src/SumoCoders/FrameworkCoreBundle/Mail/MessageFactory.php

```diff
@@ -0,0 +1,249 @@
+    {
+        if ($name !== null) {
+            $this->sender = array($email => $name);
+        } else {
+            $this->sender = $email;
+        }
+    }
+
+    /**
+     * Set the default reply-to
+     *
+     * @param string      $email
+     * @param string|null $name
+     */
+    public function setDefaultReplyTo($email, $name = null)
+    {
+        if ($name !== null) {
+            $this->replyTo = array($email => $name);
+        } else {
+            $this->replyTo = $email;
+        }
+    }
+
+    /**
+     * Set the default to
+     *
+     * @param string      $email
+     * @param string|null $name
+     */
+    public function setDefaultTo($email, $name = null)
+    {
+        if ($name !== null) {
+            $this->to = array($email => $name);
+        } else {
+            $this->to = $email;
+        }
+    }
+
+    /**
+     * Create a message
+     *
+     * @param string|null $subject
+     * @param string|null $html
+     * @param string|null $alternative
+     * @return \Swift_Message
+     */
+    private function createMessage($subject = null, $html = null, $alternative = null)
+    {
+        $message = $this->createDefaultMessage();
+
+        if ($subject != '') {
+            $message->setSubject($subject);
+        }
+
+        // only plain text
+        if ($html == '' && $alternative != '') {
+            $message->setBody($alternative, 'text/plain');
+
+            return $message;
+        }
+
+        // html mail
+        if ($alternative === null) {
+            $alternative = $this->convertToPlainText($html);
+        }
+        $message->setBody(
+            $this->wrapInTemplate($html),
+            'text/html'
+        );
+        $message->addPart($alternative, 'text/plain');
+
+        return $message;
+    }
+
+    /**
+     * Create a HTML message
+     *
+     * If no alternative is provided it will be generated automatically.
+     * This is just an alias for createMessage
+     *
+     * @param string|null $subject
+     * @param string|null $html
+     * @param string|null $plainText
+     * @return \Swift_Message
+     */
+    public function createHtmlMessage($subject = null, $html = null, $plainText = null)
+    {
+        return $this->createMessage($subject, $html, $plainText);
+    }
+
+    /**
+     * Create a plain text message
+     *
+     * @param string|null $subject
+     * @param string|null $body
+     * @return \Swift_Message
+     */
+    public function createPlainTextMessage($subject = null, $body = null)
+    {
+        return $this->createMessage($subject, null, $body);
+    }
+
+    /**
+     * @return \Swift_Message
+     */
+    public function createDefaultMessage()
+    {
+        $message = \Swift_Message::newInstance();
+
+        if (!empty($this->sender)) {
+            $message->setFrom($this->sender);
+        }
+
+        if (!empty($this->replyTo)) {
+            $message->setReplyTo($this->replyTo);
```

Comment:
Scrutiniser is juist, het is de documentatie van `\Swift_Mime_SimpleMessage` die fout is.